setup.py

```diff
@@ -23,4 +23,7 @@
             "mu = mu.app.run",
         ],
     },
+    data_files = [('/etc/udev/rules.d', ['conf/90-usb-microbit.rule',]),
+                  ('/usr/share/pixmaps', ['conf/mu.png',]),
```

Comment:
instead of adding mu.png, can we link to the file in /mu/resources/images/icon.png ?